SampleApp/RoutingViewController.swift

```diff
@@ -81,7 +81,7 @@ class RoutingViewController: UIViewController, RoutingSearchDelegate {
 
```

Comment:
Not part of your code changes but I just noticed we will crash when unwrapping `routingResult` if its null